packages/kbn-alerts-ui-shared/src/alerts_search_bar/index.tsx

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { useCallback, useState } from 'react';
+import type { Query, TimeRange } from '@kbn/es-query';
+import type { SuggestionsAbstraction } from '@kbn/unified-search-plugin/public/typeahead/suggestions_component';
+import { AlertConsumers } from '@kbn/rule-data-utils';
+import { NO_INDEX_PATTERNS } from './constants';
+import { SEARCH_BAR_PLACEHOLDER } from './translations';
+import type { AlertsSearchBarProps, QueryLanguageType } from './types';
+import { useAlertDataView } from './hooks/use_alert_data_view';
+import { useRuleAADFields } from './hooks/use_rule_aad_fields';
+import { useLoadRuleTypesQuery } from './hooks/use_load_rule_types_query';
+
+const SA_ALERTS = { type: 'alerts', fields: {} } as SuggestionsAbstraction;
+
+export const AlertsSearchBar = ({
+  appName,
+  disableQueryLanguageSwitcher = false,
+  featureIds,
+  ruleTypeId,
+  query,
+  filters,
+  onQueryChange,
+  onQuerySubmit,
+  onFiltersUpdated,
+  rangeFrom,
+  rangeTo,
+  showFilterBar = false,
+  showDatePicker = true,
+  showSubmitButton = true,
+  placeholder = SEARCH_BAR_PLACEHOLDER,
+  submitOnBlur = false,
+  http,
+  toasts,
+  unifiedSearchBar,
```

Comment:
it looks like some other packages where able to get `useKibana` working like [`HERE`](https://github.com/elastic/kibana/blob/main/packages/kbn-text-based-editor/src/text_based_languages_editor.tsx#L162-L163). Maybe, we are lucky that `unifiedSearch` can be imported as a `kbn_references`.